docs/source/community/persons_of_interest.rst

```diff
@@ -138,18 +140,23 @@ Fast Fourier Transform (torch.fft)
 -  Mike Ruberry (`mruberry <https://github.com/mruberry>`__)
 -  Peter Bell (`peterbell10 <https://github.com/peterbell10>`__)
 
-CPU Performance / SIMD
-~~~~~~~~~~~~~~~~~~~~~~
+CPU Performance (Torch Inductor / MKLDNN)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  Vitaly Fedyunin (`VitalyFedyunin <https://github.com/VitalyFedyunin>`__)
 -  Mingfei Ma (`mingfeima <https://github.com/mingfeima>`__)
+-  Jiong Gong (`jgong5 <https://github.com/jgong5>`__)
+-  Xiaobing Zhang (`XiaobingSuper <https://github.com/XiaobingSuper>`__)
```

Comment:
looks like these folks were already in merge_rules.yaml's sections of "CPU ATen backend" and "CPU frontend", so they have merge access already :)